src/UserActivityFeedEventSchemas/TagCreatedEventSchema.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (conn) => {
+  const ReviewTagCreatedEventSchema = new conn.Schema({
```

Comment:
are review tags any different than the academic profile ones? or do we actually care where such tags are being created for event tracking? I'd advocate for using only one schema and provide a discriminator field, or nothing at all (if the answer for those two questions is NO)